contracts/ERC20Safe.sol

```diff
@@ -90,16 +92,10 @@ contract ERC20Safe {
         uint256 tokenSize;
         assembly {
             tokenSize := extcodesize(token)
-        }         
+        }
         require(tokenSize > 0, "ERC20: not a contract");
 
-        (bool success, bytes memory returndata) = address(token).call(data);
-        require(success, "ERC20: call failed");
-
-        if (returndata.length > 0) {
-
-            require(abi.decode(returndata, (bool)), "ERC20: operation did not succeed");
```

Comment:
Logic that verifies the returned value to be true/false must remain.
Need to have a test for this case.